app/controllers/miq_request_controller.rb

```diff
@@ -411,10 +411,10 @@ def post_install_callback
 
   def get_request_tab_type
     case @layout
-    when "miq_request_ae"                then "AutomateRequest"
-    when "miq_request_configured_system" then "MiqProvisionConfiguredSystemRequest"
-    when "miq_request_host"              then "MiqHostProvisionRequest"
-    when "miq_request_vm"                then "MiqProvisionRequest"
+    when /ae/         then "AutomateRequest"
+    when /system/     then "MiqProvisionConfiguredSystemRequest"
+    when /host/       then "MiqHostProvisionRequest"
+    when /vm|service/ then "MiqProvisionRequest"
```

Comment:
We should not be using regex here, just plain strings. This code needs to be working for exact
`@layout` values and we need to know what the actual matching values are for potential future
refactoring efforts.